common/http.go

```diff
@@ -486,7 +487,16 @@ func (c *DatabricksClient) genericQuery(ctx context.Context, method, requestURL
 	if err != nil {
 		return nil, err
 	}
-	log.Printf("[DEBUG] %s %v <- %s %s", resp.Status, c.redactedDump(body), method, request.URL.Path)
+	headers = ""
+	if c.DebugHeaders {
+		for k, v := range resp.Header {
+			headers += fmt.Sprintf("\n * %s: %s", k, onlyNBytes(strings.Join(v, ""), c.DebugTruncateBytes))
+		}
+		if len(headers) > 0 {
+			headers += "\n"
+		}
+	}
```

Comment:
can you make it into a function, as lines 455-464 are exactly the same.

Reply:
ok